controllers/apartments_controller.rb

```diff
@@ -0,0 +1,40 @@
+# show the index page
+get "/apartments" do
+  @apt_list = Apartment.all
+  erb :"apartments/index"
+end
+# show the new apt page
+get "/apartments/new" do
+  erb :"apartments/new"
+end
+# create a new apt
+put "/apartments" do
+  edit_params = params.except("_method") # remove unwanted _method key
+  edit_params.delete_if {|key, value| value == ""} # remove blank values from the hash
+  Apartment.create(edit_params)
```

Comment:
A little confused by the consistency of this controller action, If this is a create action, then you would use a `post '/apartments'` route, and then I would just change your naming to be consistent. Otherwise if this is an update action, you would find a specific apartment, save that to an instance, then call `.update` on that instance, passing in the `edit_params`